### DISCUSSION TITLE
vim working with mouse?

### DISCUSSION THREAD

**Discussion by MikeFultonDev** [zopencommunity/meta#71] [Q&A] [answered]
I am moving myself from vi to vim and overall I really like it. Lots of new features, and so colourful.

But one thing I haven't figured out how to do is to select with the mouse and then be able to _cut and paste to another application_ (in particular cut and paste from one terminal to another). 

With vi it was easy because vi didn't have any mouse support. With vim, it lets me 'yank' but I can't figure out how to copy into my Mac buffer (I'm on a Mac if it matters). 

Any experienced vim users that use a Mac that can help?

**Comment by IgorTodorovskiIBM** [comment, marked as answer]
Hmm, command-c and command-v works for me (I'm using iTerm on my Mac though).

**Reply by MikeFultonDev** [reply]
I think this is the way to go unless we find that rebuilding with the +clipboard feature does the trick.

**Comment by jshimoda9323** [comment]
To copy to the clipboard using 'yank' it sounds like we need to compile vim with the +clipboard feature, which is non-default, according to https://stackoverflow.com/questions/3961859/how-to-copy-to-clipboard-in-vim

**Comment by MikeFultonDev** [comment]
@IgorTodorovskiIBM - where can I get iTerm? Maybe that will help? I see this:

![image](https://user-images.githubusercontent.com/61100689/200381631-eee0426a-9ca0-45b7-930e-a98916cf7fea.png)

@jshimoda9323 I am using your recommended .bashrc, so maybe I need to tweak it a bit (I did change a couple things already)

![image](https://user-images.githubusercontent.com/61100689/200381973-35ee83f5-7712-462c-9af9-205573d39a28.png)


@jshimoda9323 maybe we should build with the yank +cliboard feature? Not sure what others think

**Reply by IgorTodorovskiIBM** [reply]
I downloaded it from here: https://iterm2.com/downloads.html (brew is another option).

I think a +clipboard build is worth trying

**Reply by jshimoda9323** [reply]
yeah we can try it.  I don't know if there's a downside to turning that feature on, but some linux distributions allow you to install vim without it, or with it.  (i.e Debian Linux `vim` doesn't have +clipboard, but `vim-gtk` does)

**Comment by MikeFultonDev** [comment]
I confirmed that downloading the terminal emulator @IgorTodorovskiIBM mentioned using:
```
brew install --cask iterm2
````

solves the problem.

There appears to be much more support for the clipboard integration than in the basic mac terminal
